plugins/notifications/config.js

```diff
@@ -0,0 +1,23 @@
+const { setOptions, setMenuOptions } = require("../../config/plugins");
+const defaultConfig = require("../../config/defaults");
+
+let config = defaultConfig.plugins["notifications"];
+
+module.exports.init = (options) => {
+	config = { ...config, ...options };
+};
+
+module.exports.setAndMaybeRestart = (option, value) => {
```

Comment:
I think `setAndMaybeRestart` is more explicit since the function name indicates that the difference is the restart option
`setInMenu` doesn't tell me much tbh